src/App/App.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Login from '../Pages/login.page';
 
 class App extends React.Component {
 	render() {
 		return (
-			<div>
-				<h1>Hello World</h1>
-			</div>
+			<Router>
+				<Switch>
+					<Route path='/login'>
+						<Login />
+					</Route>
+					<Route path='/'>
+						<p> hellow world </p>
+					</Route>
```

Comment:
nice work @lemoissonM .
However, it will be good if you use `exact path` and pass component as a param like:
```
<Route exact path='/login' component={Login}/>
```
`why?`
because:
- `exact path` match like this: `/^path$/` while `path` match like this `/path/` and now you see what will go wrong!
- passing path in params will help to use `ternary operators` easily, like: **<Route exact path="/login" `component={!isAuth ? Login : Home}` />**

Reply:
Right! I just created that as a test. it's not that router or component we're going to use. it's just to show that routing is working

Reply:
ya make it l;like that so that other teammates can follow up

Reply:
okey. let me do it